addShirtAction.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+require_once 'ShirtsModel.php';
+
+$db = new PDO('mysql:host=db; dbname=collector_app', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$inputtedName = $_POST['name'] ?? false;
+$inputtedSeason = $_POST['season'] ?? false;
+$inputtedType = $_POST['type'] ?? false;
+$inputtedBrand_name = $_POST['brand_name'] ?? false;
+$inputted_img_url = $_POST['img_url'] ?? false;
+
+$shirtsModel = new shirtsModel($db);
+$shirtsModel->addNewShirt($inputtedName, $inputtedSeason, $inputtedType, $inputtedBrand_name, $inputted_img_url);
```

Comment:
See my comment in the model about returning execute first. 

You should use the return value of addNewShirt(once you've done the other comment!!!) here to determine what to do.

If it's true, use the header you have below, otherwise redirect with an error message and display it on the add form.

Might be quite hard to actually trigger the error, it would only really happen if the DB went down at an odd time, or sometime typed something reallllly daft in the form - but always worth having